src/style.css

```diff
@@ -1,3 +1,73 @@
+* {
+  margin: 0;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.navigation {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
```

Comment:
those selectors are both targeting the same element - header with class `navigation`, you can remove element selector 😉